controllers/cruisecontroltask_controller.go

```diff
@@ -22,16 +22,13 @@ import (
 	"emperror.dev/errors"
 	"github.com/go-logr/logr"
 	apiErrors "k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/apimachinery/pkg/api/meta"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/event"
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
 
-	"github.com/banzaicloud/koperator/pkg/util"
-
 	kafkav1beta1 "github.com/banzaicloud/koperator/api/v1beta1"
```

Comment:
This is not part of your change, but I was wondering if we can rename this to koperatorv1beta1?

Reply:
in a separate PR